src/common/index_node_thread_pool_wrapper.cc

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2019-2023 Zilliz. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions and limitations under the License.
+
+#include "knowhere/index_node_thread_pool_wrapper.h"
+
+#include "knowhere/comp/thread_pool.h"
+#include "knowhere/index_node.h"
+
+namespace knowhere {
+
+namespace {
+
+constexpr uint32_t cuda_concurrent_size = 16;
+
+std::shared_ptr<ThreadPool>
+GlobalThreadPoolRaft() {
```

Comment:
Super weird to have RAFT keyword here

Reply:
Why you don't just create a constructor with num_thread(int) as input, and put this raft specific number into raft's code.

Reply:
Ok. Sounds reasonable